packages/web/src/modules/zoom/components/controls/guest-control.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useCallback } from 'react';
+
+import { ui } from '@application';
+
+import { ZoomContextType } from '../../context';
+
+const { Button, Form } = ui;
+
+const GuestControl = ({
+  setZoomContextState,
+}: {
+  setZoomContextState: Function;
+}) => {
+  const onSubmit = useCallback(
+    ({
+      meetingNumber,
+      password,
+    }: {
```

Comment:
Avoid adding inline types. Define an interface outside the class like `ZoomGuestData`.